src/main/java/com/hedvig/claims/web/InternalController.kt

```diff
@@ -0,0 +1,595 @@
+import com.hedvig.claims.web.dto.ClaimsSearchRequestDTO
+import com.hedvig.claims.web.dto.ClaimsSearchResultDTO
+import com.hedvig.claims.web.dto.CreateBackofficeClaimDTO
+import com.hedvig.claims.web.dto.CreateBackofficeClaimResponseDTO
+import com.hedvig.claims.web.dto.DataItemDTO
+import com.hedvig.claims.web.dto.EmployeeClaimRequestDTO
+import com.hedvig.claims.web.dto.MarkClaimFileAsDeletedDTO
+import com.hedvig.claims.web.dto.NoteDTO
+import com.hedvig.claims.web.dto.PaymentDTO
+import com.hedvig.claims.web.dto.PaymentRequestDTO
+import com.hedvig.claims.web.dto.ReserveDTO
+import com.hedvig.claims.web.dto.StartClaimAudioDTO
+import org.axonframework.commandhandling.gateway.CommandGateway
+import org.slf4j.LoggerFactory
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDateTime
+import java.util.ArrayList
+import java.util.UUID
+import java.util.stream.Stream
+
+@RestController
+@RequestMapping(value = ["/i/claims", "/_/claims"])
+class InternalController(
+    private val claimsRepository: ClaimsRepository,
+    private val commandBus: CommandGateway,
+    private val claimsQueryService: ClaimsQueryService,
+    private val meerkat: Meerkat,
+    private val memberService: MemberService,
+    private val claimFileRepository: ClaimFileRepository,
+    private val linkFileToClaimService: LinkFileToClaimService,
+    private val productPricingService: ProductPricingService,
+    private val productPricingFacade: ProductPricingFacade
+) {
+
+    private val log = LoggerFactory.getLogger(InternalController::class.java)
+
+    @PostMapping("/startClaimFromAudio")
+    fun initiateClaim(@RequestBody requestData: StartClaimAudioDTO): ResponseEntity<*> {
+        val uuid = UUID.randomUUID()
+        val activeContracts = productPricingFacade.getActiveContracts(requestData.userId)
+        if (activeContracts.size == 1) {
+            commandBus.sendAndWait<Any>(
+                CreateClaimCommand(
+                    uuid.toString(),
+                    requestData.userId,
+                    requestData.audioURL,
+                    activeContracts[0].id
+                )
+            )
+        } else {
+            commandBus.sendAndWait<Any>(
+                CreateClaimCommand(
+                    uuid.toString(),
+                    requestData.userId,
+                    requestData.audioURL,
+                    null
+                )
+            )
+        }
```

Comment:
Maybe we feel taht this is out of scoope for this pr, and that is totally fine! Please ignore this then! But we do this  in two places in this class and doing it like this instead is cleaner to me :slightly_smiling_face: 
```suggestion
            commandBus.sendAndWait<Any>(
                CreateClaimCommand(
                    uuid.toString(),
                    requestData.userId,
                    requestData.audioURL,
                    if (activeContracts.size == 1) activeContracts[0].id else null
                )
            )
```

Reply:
Hah! I was actually doing this locally, but wanted to write a test for it first. However I didn't reeeaally wrap my head around how the tests worked so I decided against it ^^

But you're right I'll just fix this.